azext_iot/tests/test_constants.py

```diff
@@ -44,3 +44,81 @@ class FileNames:
     central_query_response_file = "central/json/query_response.json"
     central_destination_file = "central/json/destination.json"
     central_export_file = "central/json/export.json"
+
+class Certificates:
+    # BEGIN CERTIFICATE + base64 content + END CERTIFICATE
+    certificate_scenario_one = """-----BEGIN CERTIFICATE-----
+        MIIC0DCCAbgCAQAwSDELMAkGA1UEBhMCVVMxEzARBgNVBAgMCldhc2hpbmd0b24x
```

Comment:
you can generate certificates using the commands I made in _generate_root_certificate. That way you aren't storing sensitive data even though it never will be used.

Reply:
thanks will do!